app/models/broadcast.rb

```diff
@@ -10,6 +10,8 @@ class Broadcast < ApplicationRecord
   validates :banner_style, inclusion: { in: VALID_BANNER_STYLES }, allow_blank: true
   validate  :single_active_announcement_broadcast
 
+  before_save :update_last_active_at, if: :will_save_change_to_active?
```

Comment:
This still feels a little confusing to me, sometimes this time is when the broadcast was last active(when going from active to inactive) and sometimes this time is when the broadcast was last activated(when going from inactive to active).  What if instead, we called this timestamp `active_status_updated_at` ?

Reply:
I'm happy to change the name! For clarification purposes on my end though, when you refer to "timestamp" here, is it the timestamp on the Broadcast table (`last_active_at`) or the act of updating the timestamp in `#update_last_active_at` (asking because we're in the model now and I want to ensure I understand!) @mstruve.

Reply:
Good question, I'm talking about the column name in the database.

Reply:
Perfect; I'll make that change! 😄